CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Change History
 **************
 
+2.13.x (unreleased)
+===================
+
+- Fix bootstrapping for python27
+
+
 3.0.0a3 (unreleased)
```

Comment:
I object to non-monotonic version numbers in the changelog.  If we want to backport this to the 2.x release, we probably want to do that n a branch with a changelog that doesn't have all these 3.0.0 alpha versions.

Reply:
Very well. Makes sense. Will be fixed with the next commit.